cmd/mothership/local/options.go

```diff
@@ -52,18 +52,10 @@ func componentsFromFile(path string) ([]string, []string, error) {
 
 	for _, c := range compList.Prerequisites {
 		preComps = append(preComps, c.Name)
-		if c.Namespace != "" {
-			defaultComps = append(defaultComps, c.Name+"@"+c.Namespace)
-		} else {
-			defaultComps = append(defaultComps, c.Name)
-		}
+		defaultComps = append(defaultComps, fmt.Sprintf("{%s,%s,%s}", c.Name, c.Namespace, c.URL))
```

Comment:
All empty no make sens, because if we don't get the name of component we don't recognise component. 

Empty namespace and/or url is ok. In this case reconciler get default value. 
[test](https://github.com/kyma-incubator/reconciler/blob/acb15111c7922b6e11dea184b5e1db2475174de1/cmd/mothership/local/local_test.go#L10)